### ISSUE TITLE
connect meal calendar data from our fetch plugin

### ISSUE THREAD

**Issue by atherdon** [GroceriStar/react-only-intern-23#72]
this task is not ready, because we have a lot of blockers
GroceriStar/types#10

**Comment by atherdon** [comment]
and we should replace the calendar items data. And i think we can remove them.
Or keep it, because we plan to move away our components

**Comment by sanchit94** [comment]
Yeah, I think we should keep them as it is for now as we are planning to shift them to another repo. Also I think it will be a good exercise for the intern, or maybe if it is a priority I'll do it! :)

**Comment by atherdon** [comment]
to be honest - i was a bit crazy yesterday - and started to move it to a separated folder/repo myself. Wanted to learn how to use bitsrc.

And it was not an easy for me :) So i think a new members will be driven crazy with this kind of task. 

Still - we'll use our plan from #70 but maybe related to another work. Because making a standalone components it's not an easy thing. But it's cool to see that this is a good idea. because right now it's completely ruined. if you want - i can send you that repo, so you can take a look.
So when it'll be done - it'll sky rocket the code quality

**Comment by sanchit94** [comment]
Yea sure send me the repo I can have a quick look at it!

**Comment by atherdon** [comment]
https://github.com/GroceriStar/calendar

**Comment by sanchit94** [comment]
No issues I will make it work on that repo :)
